app/api/schema/video_stream.py

```diff
@@ -42,3 +42,12 @@ class Meta:
         schema='VideoChannelSchemaPublic',
         type_='video-channel',
     )
+    video_stream_moderator = Relationship(
```

Comment:
First of all, again, this should be plural. There are multiple moderators of a video stream. It is many to many. You have written many=True as well

Secondly, the backref is moderators, so this should be `moderators`